lang/dart.md

```diff
@@ -0,0 +1,206 @@
+
+Edit environment variable window appears. Click on New and paste the dart sdk bin path as shown below.
+
+**Step 4:** Restart Command Prompt
+Close the existing command prompt window and open a new command prompt.
+
+Just run the dart command:  `dart --version`
+
+
+**Install the Dart SDK on Linux:**
+
+
+The steps of Dart installation on Linux is given below.
+
+Before installing the Dart, if you are Debian/Ubuntu on AMD64(64-bit Intel) in your local machine, you can install the Dart through one of the following options.
+
+Install using apt-get
+Install a Debian package
+Installation using apt-get
+
+**Step -1:**
+
+ Type the following commands for a one-time setup.
+
+$sudo apt-get update  
+$ sudo apt-get install apt-transport-https  
+  
+$ sudo sh -c 'wget -qO- https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add -'  
+  
+$ sudo sh -c 'wget -qO- https://storage.googleapis.com/download.dartlang.org/linux/debian/dart_stable.list > /etc/apt/sources.list.d/dart_stable.list'  
+**Step - 2:**
+
+ Type the following command in terminal to install the Dart SDK using apt-get option.
+
+$sudo apt-get update  
+$ sudo apt-get install dart  
+It will successfully download the Dart SDK.
+
+
+**Installation a Debian Package:**
+
+We can download Dart SDK as a Debian package in the .deb package format. To make all Dart binaries accessible, we have to change the PATH by typing the following command.
+
+export PATH="$PATH:/usr/lib/dart/bin"  
+To change the PATH for upcoming terminal sessions, use a below command:
+
+$ echo 'export PATH="$PATH:/usr/lib/dart/bin"' >> ~/.profile  
+
+
+## Popular Dart IDEs/editors 
+- The Eclipse,VsCode, IntelliJ, and WebStorm are the IDEs from the Jet brains that support the Dart Programming, but WebStorm is more popular than others. We can download it from https://www.jetbrains.com/webstorm/download/#section=windows-version.
+
+
+## Some Cool projects in Dart
+
+- **1. [Invoice Naija](https://github.com/invoiceninja/admin-portal) :** This is a business application for documenting invoices, tasks, and expenses.
+It was built using flutter and laravel.
+
+- Invoice Naija
+- Extra Features
+- Clients
+- Products
+- Quotes
+- Payments
+ 
+
+**2. [Bit Alarm](https://github.com/damoonrashidi/bitalarm) :** This is a cryptocurrency application created to keep track of different cryptocurrencies
+
+Bit Alarm
+Extra Features
+Loading indicator.
+ Graph out historical data for a currency
+Scan QR-code to add a wallet to wallet list
+ Remove wallet from the list
+ Dynamic portfolio based on the address (ETH + ERC20-tokens, LTC, BTC, BCH, DASH and ADA for now)
+ Further, Ability to add individual assets in addition to wallets
+
+**3 [Weight Tracker](https://github.com/MSzalek-Mobile/weight_tracker) :** Weight Tracker is an application dedicated to people who want to dump or maintain weight. It allows you to keep an eye on your progress and pursue your weight goals.
+
+- **Screenshot Image**
+
+- **Extra Features**
+
```

Comment:
Remove newlines in between here. As they won't be rendered anyways. Remove to improve readability.